### DISCUSSION TITLE
kiss-icp install

### DISCUSSION THREAD

**Discussion by Sefilevy** [PRBonn/kiss-icp#335] [Q&A]
we try to build it on ROS2 by running the code:
"git clone https://github.com/PRBonn/kiss-icp
colcon build
source ./install/setup.bash"
we got the following error:
" colcon build
[0.892s] ERROR:colcon:colcon build: Duplicate package names not supported:
- kiss_icp:
  - kiss-icp/ros
  - src/kiss-icp/ros"
 
if anyone can help us- much appreciated

**Comment by nachovizzo** [comment]
Closing this as duplicate of #336
